src/scrapers/pages/video.ts

```diff
@@ -88,6 +89,8 @@ export function parseByDom(html: string, $: CheerioAPI) {
     const durationMeta = $('head meta[property="video:duration"]')
     const duration = +getAttribute<number>(durationMeta, 'content', 0)
     const durationFormatted = toHHMMSS(duration)
+    const ldPlusJson = $('head script[type="application/ld+json"]')
```

Comment:
Thanks for this! Can we move it to another PR?

Reply:
Created #76